internal/provisioner/utility_group.go

```diff
@@ -65,6 +65,7 @@ var helmRepos = map[string]string{
 	"ingress-nginx":        "https://kubernetes.github.io/ingress-nginx",
 	"prometheus-community": "https://prometheus-community.github.io/helm-charts",
 	"bitnami":              "https://charts.bitnami.com/bitnami",
+	"fluent":               "https://fluent.github.io/helm-charts",
```

Comment:
@angeloskyratzakos maybe remove stable if we don't need it anymore? Can you check if its needed anywhere else?

Reply:
You are right! removing it as it is not needed any more.